webui/src/Components/ConfigurationForm.tsx

```diff
@@ -272,8 +272,16 @@ const ConfigurationForm = () => {
     const result = recursiveReplace(structuredClone(schema))
     const formData = new FormData(e.target as HTMLFormElement)
     const formDataJson = Object.fromEntries(formData.entries())
-    Object.entries(formDataJson).forEach(([key, value]) => {
+    Object.entries(formDataJson).forEach(([key, _value]) => {
       const schemaEntry = jp.query(schema, key)
+      let value = _value
+      if (value === '') {
+        if (schemaEntry.length > 0 && schemaEntry[0]['type'] === 'nullOr') {
+          value = null
```

Comment:
Don't know much about the code but I get the following error when trying to run `yarn build`:
```shell
[kari@torque:~/Workspace/HomestakerOS]$ yarn build
yarn run v1.22.19
$ tsc -p webui/tsconfig.json && vite build webui
webui/src/Components/ConfigurationForm.tsx:280:11 - error TS2322: Type 'null' is not assignable to type 'FormDataEntryValue'.

280           value = null
              ~~~~~


Found 1 error in webui/src/Components/ConfigurationForm.tsx:280

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
my bad, too hasty

Reply:
no problem

Reply:
that should work now